conans/client/graph/graph_manager.py

```diff
@@ -294,15 +300,15 @@ def _recurse_build_requires(self, graph, builder, check_updates,
                 continue
             # Packages with PACKAGE_ID_UNKNOWN might be built in the future, need build requires
             if (node.binary not in (BINARY_BUILD, BINARY_EDITABLE, BINARY_UNKNOWN)
-                    and node.recipe != RECIPE_CONSUMER):
+                and node.recipe != RECIPE_CONSUMER):
```

Comment:
Can we put this in EditorConfig? My configuration complains this is bad and needs an indent.